index.html

```diff
@@ -162,7 +162,9 @@ <h3 class="participant-name">Stefan Bache</h3>
   </div>
   <div class="further-col">
     <div class="participant-row">
-        <a href="http://ropensci.org/unconf_nomination">Please nominate yourself if you'd like to attend</a>
+        <a href="http://ropensci.org/unconf_nomination">Please nominate yourself if you'd like to attend</a>. 
+
+        <br><br><i>By design, the unconference works best as a small event with ~50-60 attendees. A selection committee goes through a list of suggested names along with these nominations to pick a combination of people that are representatives of a variety of career stages, fields, and gender and geographic diversity. Approximately a third of the attendees return in consecutive years, with most of the attendees being first time participants.</i>
```

Comment:
* "selection committee _will go_ through a list..."
* "are _representative_ of" (drop s)
* "ages, fields, _genders and locations_" (perhaps? current version reads oddly but can't immediately think of better)